mypy.ini

```diff
@@ -41,8 +41,10 @@ disallow_untyped_calls = true
 disallow_untyped_decorators = true
 disallow_untyped_defs = true
 no_implicit_optional = true
+strict_equality = true
```

Comment:
I don't know much about mypy configuration in Home Assistant, but it looks like this PR touches the configuration for a lot of other components as well. Is this just an accidental change?